kong/clustering/config_helper.lua

```diff
@@ -0,0 +1,232 @@
+local constants = require("kong.constants")
+local declarative = require("kong.db.declarative")
+
+local tostring = tostring
+local assert = assert
+local type = type
+local error = error
+local pairs = pairs
+local ipairs = ipairs
+local concat = table.concat
+local sort = table.sort
+
+local isempty = require("table.isempty")
+local isarray = require("table.isarray")
+local nkeys = require("table.nkeys")
+local new_tab = require("table.new")
+
+local ngx_log = ngx.log
+local ngx_null = ngx.null
+local ngx_md5 = ngx.md5
+local ngx_md5_bin = ngx.md5_bin
+
+--local ngx_ERR = ngx.ERR
```

Comment:
We should remove these comments that are not needed.

Reply:
done